includes/discovery/storage/eql-storage.inc.php

```diff
@@ -0,0 +1,37 @@
+<?php
+
+/*
+ * LibreNMS
+ *
+ * Copyright (c) 2016 Peter TKATCHENKO https://github.com/Peter2121/
+ *
+ * This program is free software: you can redistribute it and/or modify it
+ * under the terms of the GNU General Public License as published by the
+ * Free Software Foundation, either version 3 of the License, or (at your
+ * option) any later version.  Please see LICENSE.txt at the top level of
+ * the source code distribution for details.
+ */
+
+$eql_storage = snmpwalk_cache_oid($device, 'EqliscsiVolumeEntry', null, 'EQLVOLUME-MIB', $config['install_dir'].'/mibs/equallogic');
```

Comment:
Think you need to check if the os is equallogic first otherwise this will run for all devices.